### PR TITLE
BLD: enable SSE2 for 32-bit msvc 9 and 10 compilers

### PR DESCRIPTION
**Pull request by cgohlke**
Fixes https://github.com/numpy/numpy/issues/6428

### CODE REVIEW

**Comment by charris** [comment]
Also marking this for 1.10.1 release consideration, but will wait to see if it is appropriate.

File: numpy/distutils/msvccompiler.py [hunk 1]
```diff
@@ -15,3 +17,6 @@ def initialize(self, plat_name=None):
             os.environ['lib'] = environ_lib + os.environ['lib']
         if environ_include is not None:
             os.environ['include'] = environ_include + os.environ['include']
+        if platform_bits == 32:
```
Comment (charris) [inline numpy/distutils/msvccompiler.py]: A note on why SSE2 is reguired would be helpful. Is there anyway to make this specific to compiler versions <= 1500?

Comment (juliantaylor) [inline numpy/distutils/msvccompiler.py]: msvc will use the sse2 code anyway, this just makes it consistent over all of numpy

**Comment by charris** [comment]
Added comment in #6444, so closing this. Thanks Christoph.

**Pull request closed without merging**